velox/dwio/catalog/fbhive/FileUtils.cpp

```diff
@@ -211,7 +218,7 @@ std::vector<std::pair<std::string, std::string>> FileUtils::parsePartKeyValues(
 
 std::string FileUtils::extractPartitionName(const std::string& filePath) {
   const auto& partitionParts = extractPartitionKeyValues(filePath);
-  return partitionParts.empty() ? "" : makePartName(partitionParts);
+  return partitionParts.empty() ? "" : makePartName(partitionParts, true);
```

Comment:
Shouldn't this be 'false'? Don't we want to extract partition name from the file path as is?

Reply:
@mbasmanova Update to false.